Tests/SwiftProtobufTests/Test_Struct.swift

```diff
@@ -288,18 +281,6 @@ class Test_JSON_Value: XCTestCase, PBTestHelpers {
     }
 
     func testValue_list() throws {
-        XCTAssertEqual(Google_Protobuf_ListValue(any: [1]),
```

Comment:
Ah yes, I forgot to mention this. Once I moved the convenience initializers to extensions, I started getting an odd ambiguity error for the `Any` ones. I ended up thinking about them some more and decided that they were probably not worth the potential problems they add—the fact that they reduce type safety and silently ignore values that don't match any of the allowed `Value` kinds seemed like it could cause problems that users would have difficulty tracking down.